Azure-Samples/AnomalyDetector
# Support Azure Machine Learning Studio

Change to adapt this code to run on Azure Machine Learning Notebook as well

## Purpose
 
* This change provides support of this code to run on Azure Machine Learning Studio while continuing to run on vanilla Jupyter Notebook.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Run the existing ipynb file in AML Studio and it will fail to get the correct location with this code -->

```
 new_dir = os.path.join('.', str(uuid.uuid1()))
```

*  Now use the updated ipynb file which has the following code instead and it will run in in AML Studio successfully.
```
    cwd = os.getcwd()
    new_dir = os.path.join(cwd, str(uuid.uuid1()))

```

* Running the following code should not throw any errors after the change has been made

```
data_source = SOURCE_BLOB_SAS
local_data_path = "sample_data_5_3000.zip"
severity = 0.7
start_date = "2021-01-02T12:00:00Z"
end_date = "2021-01-03T00:00:00Z"
series, raw_result, top_anomaly = draw(data_source, local_data_path, result_id, severity, start_date, end_date)
```

## What to Check
Verify that the following are valid
* You have the correct values for the variables ENDPOINT and HEADERS.

## Other Information
